site/docs/resources/faq.md

```diff
@@ -115,6 +115,14 @@ There are a few things you can do.
 4. Come ask us in the group chat for help. We have experienced people there.
 5. It is possible to ask Telegram to increase the limits, but this is very unlikely to happen if you did not do steps 1-3 first.
 
+### Cannot find type definition file for 'node-fetch'.
```

Comment:
According to https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md026

We should remove this punctuation.

Reply:
I agree.
```suggestion
### Cannot find type definition file for 'node-fetch'
```

Reply:
Or this is the orignal error message which we should keep it what it is?

Reply:
It was the original message, but I think you can remove the dot. The meaning will stay there.